Makefile

```diff
@@ -1,16 +1,16 @@
 all:
 
 debug:
-	@node_modules/.bin/node-debug fathom.js
+	@npm run debug
 
 lint:
-	@node_modules/.bin/jshint fathom.js
+	@npm run lint
 
 test:
-	@node_modules/.bin/mocha
+	@npm run test
 
 debugtest:
-	@node_modules/.bin/mocha -d
+	@npm run test -- -d
```

Comment:
Not sure if the Makefile adds anything here that we can't achieve with just the npm scripts tasks.
But I'm just a drive-by person walking in here and kicking up dust.